core/variables.js

```diff
@@ -493,9 +489,6 @@ const getVariable = function(workspace, id, opt_name, opt_type) {
     }
     // Otherwise look up by name and type.
     variable = workspace.getVariable(opt_name, opt_type);
```

Comment:
We would need to do the same kind of logic here to check the target workspace first.

Reply:
(Unless we move the code to `WorkspaceSvg.prototype.getVariableById` as I propose above.)